flask_app/blueprints/api/replication.py

```diff
@@ -8,11 +8,11 @@
 NoneType = type(None)
 
 
-@API
+@API(version=2)
 @requires_role('admin')
-def create_replication(url: str, username: (str, NoneType)=None, password: (str, NoneType)=None):
+def create_replication(url: str, index_name: str='backslash', username: (str, NoneType)=None, password: (str, NoneType)=None):
```

Comment:
This restores the change in my previous comment - the API *forces* you to pass the index name, and old clients would break. The correct change here is to make the default NULL, and enforce the default through a constant with an explicit test for `None`

Reply:
the fact that index_name gets default value doesn't mean that you can omit passing it? when i ran locally test_replications.py without giving the index_name='bla' it passed..

Reply:
Oh - I'm stupid. Ignore me.